wd_helper.au3

```diff
@@ -1395,13 +1396,12 @@ Func _WD_UpdateDriver($sBrowser, $sInstallDir = Default, $bFlag64 = Default, $bF
 						__WD_CloseDriver($sDriverEXE)
 						FileDelete($sInstallDir & $sDriverEXE)
 
+						#Region - Extract new instance of webdriver
 						; Handle COM Errors
 						Local $oErr = ObjEvent("AutoIt.Error", __WD_ErrHnd)
 						#forceref $oErr
-
-						; Extract new instance of webdriver
 						$oShell = ObjCreate("Shell.Application")
-						If @error Then
+						If @error Or FileGetSize($sTempFile) = 0 Or IsObj($oShell.NameSpace($sTempFile)) = 0 Then
 							$iErr = $_WD_ERROR_GeneralError
```

Comment:
`$_WD_ERROR_GeneralError` is used in two places. 
Here in `_WD_UpdateDriver` and internally in `_WD_Download`

Using a different error code for one of these might help with future troubleshooting efforts.

Reply:
also 
`FileGetSize($sTempFile) = 0 Or IsObj($oShell.NameSpace($sTempFile)) = 0`
should fire error like `$_WD_ERROR_FileIssue`

Reply:
To track errors we can use 
`__WD_Error($sFuncName, $_WD_ERROR_Exception, 'some descriptions')`
in additionall places.

Reply:
Can we cosider this part as "Resolved conversation" ?